user/installing-dependencies.md

```diff
@@ -12,14 +12,18 @@ redirect_from:
 
 To install Ubuntu packages that are not included in the default [standard](/user/reference/precise/), use apt-get in the `before_install` step of your `.travis.yml`:
 
+By default, `apt-get update` does not get run automatically unless you are using the [APT addon](#Installing-Packages-with-the-APT-Addon). If you want to update `apt-get` automatically on every build, add something like the following to your `.travis.yml`:
+
 ```yaml
 before_install:
-  - sudo apt-get -qq update
   - sudo apt-get install -y libxml2-dev
+addons:
```

Comment:
I think we should split this into two examples.

```yaml
before_install:
  - sudo apt-get install -y libxml2-dev
```

and

```yaml
addons:
  apt:
    update: true
before_install:
  - sudo ./install_meta_dependencies # a custom script which invokes `sudo apt-get install`, but not `sudo apt-get update`
```

Aside: I can't think of a compelling use case of `addons.apt.update: true`; the user can easily add `sudo apt-get update` into the custom script if needed.